test/unit/utglTF2ImportExport.cpp

```diff
@@ -753,12 +753,21 @@ TEST_F(utglTF2ImportExport, wrongTypes) {
     // Deliberately broken version of the BoxTextured.gltf asset.
     using tup_T = std::tuple<std::string, std::string, std::string, std::string>;
     std::vector<tup_T> wrongTypes = {
```

Comment:
Out of curiosity, if you make an alias for `std::vector<tup_T>`, does this also solve the compiler error?

Reply:
Thanks for your suggestion. I will try it and feedback later.